paasta_tools/setup_marathon_job.py

```diff
@@ -249,7 +249,8 @@ def log_bounce_action(line, level='debug'):
         if drain_method.is_safe_to_kill(task):
             killed_tasks.add(task)
             log_bounce_action(line='%s bounce killing drained task %s' % (bounce_method, task.id))
-            marathon_tools.kill_task(client=client, app_id=task.app_id, task_id=task.id, scale=True)
+
+    client.kill_given_tasks(task_ids=[task.id for task in killed_tasks], scale=True)
```

Comment:
This kinda reads weird now. Can you simplify the code and the code above (:248) and call this variable "tasks_to_kill"?